### PR TITLE
feat: add carousel for wallet

### PR DESCRIPTION
**Pull request by alonakaliuzhna**
### Description
it is ready for review

Please provide a brief description of the changes you have made in your PR.

### Type of change

- [x] Feature (introduces new functionality)
- [ ] Bugfix
- [ ] Configuration
- [x] Refactoring (doesn't change functionality, only the code)
- [x] Style changes (doesn't affect functionality, only the look of the components)

### PR Checklist

- [x] Synced up with latest from the main branch/Code is linted
- [x] Description of the changes has been added (Screenshots/Video)
- [x] Chakra components/constants/hooks have been used wherever possible

> If not explain here

- [x] Commit messages follow [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary)
- [x] Console has been checked for warnings

#### Screenshots/Video before change

Add screenshots from the [current deployed state](https://sandbox-money-app.herokuapp.com/) of the changed components here.

#### Screenshots/Video after change

Add screenshots with the changes applied here.
65a4cc3ad31befccab2abf18cd3d0896

https://user-images.githubusercontent.com/83176493/183087886-4c0060bd-386f-477e-9786-1e4a1777500b.mp4

### CODE REVIEW

**Review by kalashnikau — DISMISSED** [review]

**Review by PavelTitov16 — COMMENTED** [review]
maybe better to do carousel with overlay to show only 1 wallet? and by clicking arrow button to swipe one  after the other and hide others

File: src/components/WalletCarousel/WalletCarousel.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        width="500px"
+        showThumbs={false}
+        showStatus={false}
+        centerMode={true}
+        centerSlidePercentage={30}
+        showIndicators={false}
+        infiniteLoop={true}
+        renderArrowPrev={(onClickHandler, hasPrev, label) =>
+          hasPrev && (
+            <button
```
Comment (itsnotpolza) [inline src/components/WalletCarousel/WalletCarousel.jsx:23]: why not use IconButton from chakra?

File: src/components/WalletCarousel/WalletCarousel.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                cursor: 'pointer'
+              }}
+            >
+              <Icon as={ArrowBackIcon} />
+            </button>
+          )
+        }
+        renderArrowNext={(onClickHandler, hasNext, label) =>
+          hasNext && (
+            <button
```
Comment (itsnotpolza) [inline src/components/WalletCarousel/WalletCarousel.jsx:43]: here as well

File: src/components/WalletCarousel/WalletCarousel.jsx (hunk 1 above)
Comment (alonakaliuzhna) [reply src/components/WalletCarousel/WalletCarousel.jsx]: Dond

File: src/components/WalletCarousel/WalletCarousel.jsx (hunk 2 above)
Comment (alonakaliuzhna) [reply src/components/WalletCarousel/WalletCarousel.jsx]: Done

**Review by kalashnikau — DISMISSED** [review]

**Review by itsnotpolza — DISMISSED** [review]

**Review by lazzzaroni — DISMISSED** [review]

**Review by itsnotpolza — APPROVED** [review]

**Review by lazzzaroni — APPROVED** [review]

**Pull request merged**